driver-core/src/main/resources/META-INF/native-image/org.mongodb/bson/native-image.properties

```diff
@@ -13,4 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-Args = --initialize-at-run-time=com.mongodb.UnixServerAddress,com.mongodb.internal.connection.SnappyCompressor,org.bson.types.ObjectId,com.mongodb.internal.connection.ClientMetadataHelper
+Args =\
+  --initialize-at-run-time=\
+    com.mongodb.UnixServerAddress,\
+    com.mongodb.internal.connection.SnappyCompressor,\
+    com.mongodb.internal.connection.ClientMetadataHelper,\
```

Comment:
I checked by looking at the reports generated by `native-image` that these `\` line breaks work fine.